src/slack/tests/resources/test.txt

```diff
@@ -0,0 +1 @@
+This is a file used in tests
```

Comment:
Do we need this?

Reply:
Yes we are using it in the tests as stated.